components/scripts/gradle/03-validate-local-build-caching-different-locations.sh

```diff
@@ -320,20 +320,12 @@ explain_and_print_summary() {
   read_build_scan_metadata
   local text
   IFS='' read -r -d '' text <<EOF
-The ‘Summary’ section below captures the configuration of the experiment and the
-two build scans that were published as part of running the experiment. The build
-scan of the second build is particularly interesting since this is where you can
-inspect what tasks were not leveraging the local build cache.
+$(explain_summary)
 
 The ‘Build Caching Leverage’ section below reveals the realized and potential
 savings from build caching. All cacheable tasks' outputs need to be taken from
 the build cache in the second build for the build to be fully cacheable.
-
-The ‘Investigation Quick Links’ section below allows quick navigation to the
-most relevant views in build scans to investigate what tasks were avoided due to
-local build caching and what tasks executed in the second build, which of those
-tasks had the biggest impact on build performance, and what caused those tasks
-to not be taken from the local build cache.
+$(explain_quick_links)
```

Comment:
Dropping the `Investigation Quick Links` section causes there to be two new lines between the `Build Caching Leverage` and `Command Line Invocation` sections when running with `--no-build-scan-publishing --interactive`.

Reply:
Yes indeed.  I'm still working on it. It is proving to be surprisingly difficult to fix.

Reply:
✅ This is fixed now. I ended up changing the function structure to avoid this problem entirely.